src/main/config/parameter_group_ids.h

```diff
@@ -67,9 +67,12 @@
 #define PG_SPECIAL_COLOR_CONFIG 46  // part of led strip, structs OK
 #define PG_PILOT_CONFIG 47 // does not exist in betaflight
 #define PG_MSP_SERVER_CONFIG 48 // does not exist in betaflight
-#define PG_VOLTAGE_METER_CONFIG 49 // Cleanflight has voltageMeterConfig_t, betaflight has batteryConfig_t
-#define PG_AMPERAGE_METER_CONFIG 50 // Cleanflight has amperageMeterConfig_t, betaflight has batteryConfig_t
+#define PG_VOLTAGE_SENSOR_ADC_CONFIG 49 // renamed from PG_VOLTAGE_METER_CONFIG
+#define PG_CURRENT_SENSOR_ADC_OR_VIRTUAL_CONFIG 50 // renamed from PG_AMPERAGE_METER_CONFIG
```

Comment:
Actually, I think current is used far too often in the temporal sense. eg `currentPidProfile` `blackboxCurrent` etc. These should just be `pidProfile` and `blackbox`.

The `current` prefix is used far too often and can be dropped in almost all cases, off the top of my head the only one I would leave is `currentTime`. Once that is done the ambiguity disappears.